api/types/types.go

```diff
@@ -185,6 +185,9 @@ type Version struct {
 type Info struct {
 	ID                 string
 	Containers         int
+	ContainersRunning  int
+	ContainersPaused   int
+	ContainersStopped  int
```

Comment:
On second thought, Total might be unnecessary as one could get the total number adding Running and Stopped.